sdk/sdktypes/value_custom.go

```diff
@@ -0,0 +1,65 @@
+package sdktypes
+
+import (
+	"errors"
+
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	valuev1 "go.autokitteh.dev/autokitteh/proto/gen/go/autokitteh/values/v1"
+)
+
+type CustomValuePB = valuev1.Custom
+
+type customValueTraits struct{}
+
+func (customValueTraits) Validate(m *CustomValuePB) error {
+	return errors.Join(
+		executorIDField("executor_id", m.ExecutorId),
+		objectField[Value]("value", m.Value),
+	)
+}
+
+func (customValueTraits) StrictValidate(m *CustomValuePB) error {
+	return errors.Join(
+		mandatory("executor_id", m.ExecutorId),
+	)
+}
+
+var _ objectTraits[*CustomValuePB] = customValueTraits{}
```

Comment:
I general I don't like this style. When testing the compiler will fail if the interface is not implemented.